pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -299,9 +299,9 @@ def evaluate(self, model, dataloaders, max_batches, test=False):
 
         return eval_results
 
-    def run_evaluation(self, test=False):
+    def run_evaluation(self, test_mode: bool = False):
         # when testing make sure user defined a test step
-        if test and not (self.is_overriden('test_step') or self.is_overriden('test_end')):
+        if test_mode and not (self.is_overriden('test_step') or self.is_overriden('test_end')):
```

Comment:
We should make sure after we merge #938 into master that we update the pr here so that auto-merge doesn't reset this line again.